src/scripts/main.js

```diff
@@ -1 +1,67 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+new Promise((resolve, reject) => {
+  body.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+}).then(data => {
```

Comment:
move it to const and after add then and catch 
(with variable you can reuse if needed )

Reply:
done!